### ISSUE TITLE
VirtualBox + Mojave Version Issue

### ISSUE THREAD

**Issue by triplingual** [dnoneill/annotate#5]
tl;dr: I think you need to specify VirtualBox version 6.0.x as the dependency rather than just VirtualBox.

Longer explanation:
VirtualBox's latest version as of this issue date is 6.1.2, but is not compatible with Vagrant 2.2.6, the latest as of this issue date.
Vagrant's error, when trying to force using VirtualBox by running `vagrant up --provider=virtualbox` is 
```
Vagrant has detected that you have a version of VirtualBox installed
that is not supported by this version of Vagrant. Please install one of
the supported versions listed below to use Vagrant:

4.0, 4.1, 4.2, 4.3, 5.0, 5.1, 5.2, 6.0
```

This kinda sucks, as VirtualBox says the 6.0.x. line is only expected to be supported through July 2020.

On the other hand, there's [a GitHub Issue](https://github.com/oracle/vagrant-boxes/issues/178) that shows how to ~hack~ edit some Ruby files of Vagrant and make it work. Requires more experience/skills/confidence than using the older version of VirtualBox, though.

Apologies if I'm missing something that makes this issue irrelevant or wrong.

_Edit 00: Neglected to link the Oracle blog post, but found a GitHub issue in the meantime anyway, and linked that._

**Comment by dnoneill** [comment]
@triplingual Thanks so much added it to README and website.
